bundles/org.openhab.binding.icalendar/README.md

```diff
@@ -23,20 +25,54 @@ Each `calendar` thing requires the following configuration parameters:
+|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------|
+| `maxEvents`      | The count of expected results.                                                                                                                                                               | mandatory                                  |
+| `refreshTime`    | The frequency in minutes the channels get refreshed.                                                                                                                                         | mandatory (default available)              |
+| `datetimeUnit`   | A unit for time settings in this filter. Valid values: `MINUTE`, `HOUR`, `DAY` and `WEEK`.                                                                                                   | optional/required for time-based filtering |
+| `datetimeStart`  | The start of the time frame where to search for events relative to current time. Combined with `datetimeUnit`.                                                                               | optional                                   |
+| `datetimeEnd`    | The end of the time frame where to search for events relative to current time. Combined with `datetimeUnit`. The value must be greater than `datetimeStart` to get results.                  | optional                                   |
+| `datetimeRound`  | Whether to round the datetimes of start and end down to the earlier time unit. Example if set: current time is 13:00, timeunit is set to `DAY`. Resulting search will start and end at 0:00. | optional                                   |
+| `textEventField` | A field to filter the events text-based. Valid values: `SUMMARY`, `DESCRIPTION`, `COMMENT`, `CONTACT` and `LOCATION` (as described in RFC 5545).                                             | optional/required for text-based filtering |
+| `textEventValue` | The text to filter events with.                                                                                                                                                              | optional                                   |
+| `textValueType`  | The type of the text to filter with. Valid values: `TEXT` (field must contain value), `REGEX` (field must match value, completely, dot matches all, case insensetive).                       | optional/required for text-based filtering |
+
 ## Channels
 
-The channels describe the current and the next forthcoming event.
+The channels of `calendar` describe the current and the next forthcoming event.
```

Comment:
Ditto sub title ###Channels for "calendar"

Reply:
OK.